### PR TITLE
Renamed app->service

### PR DESCRIPTION
**Pull request by ewoutp**
This change renames internal variables and functions from app to service.

This makes sure that all functions and variables used in the code match with the vocabular we use.

After this PR, there are still "App" names in here coming from both user-config and client-go.

### CODE REVIEW

**Comment by ewoutp** [comment]
@zyndiecate RFR

**Comment by ewoutp** [comment]
ping @denderello

File: cat.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ const (
 func catCmdRun(cmd *cobra.Command, args []string) {
 	switch len(args) {
 	case 0:
-		appName, err := getAppNameFromDefaultConfigFile(cmd.Name(), "Cannot show the definition of your service due to a syntax error in '%s'.")
+		serviceName, err := getServiceNameFromDefaultConfigFile(cmd.Name(), "Cannot show the definition of your service due to a syntax error in '%s'.")
```
Comment (zyndiecate) [inline cat.go]: Getter without `get` prefix.

File: delete.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ func init() {
 func deleteCmdRun(cmd *cobra.Command, args []string) {
 	switch len(args) {
 	case 0:
-		appName, err := getAppNameFromDefaultConfigFile(cmd.Name(), "Cannot delete your service due to a syntax error in '%s'")
+		serviceName, err := getServiceNameFromDefaultConfigFile(cmd.Name(), "Cannot delete your service due to a syntax error in '%s'")
```
Comment (zyndiecate) [inline delete.go]: Getter without `get` prefix.

File: start.go [hunk 3]
```diff
@@ -62,7 +62,7 @@ func init() {
 func startCmdRun(cmd *cobra.Command, args []string) {
 	switch len(args) {
 	case 0:
-		appName, err := getAppNameFromDefaultConfigFile(cmd.Name(), "Cannot start your service due to a syntax error in '%s'")
+		serviceName, err := getServiceNameFromDefaultConfigFile(cmd.Name(), "Cannot start your service due to a syntax error in '%s'")
```
Comment (zyndiecate) [inline start.go]: Getter without `get` prefix.

**Comment by zyndiecate** [comment]
LGTM and WFM

File: cat.go (hunk 1 above)
Comment (ewoutp) [inline cat.go]: Done

File: delete.go (hunk 2 above)
Comment (ewoutp) [inline delete.go]: Done

File: start.go (hunk 3 above)
Comment (ewoutp) [inline start.go]: Done

**Pull request merged**